azure-quantum/tests/unit/recordings/test_job_submit_microsoft_simulated_annealing.yaml

```diff
@@ -5,15 +5,15 @@ interactions:
       Accept:
       - application/json
       Accept-Encoding:
-      - gzip, deflate, br
+      - gzip, deflate
       Connection:
       - keep-alive
       Content-Length:
       - '287'
       Content-Type:
       - application/x-www-form-urlencoded
       User-Agent:
-      - azsdk-python-identity/1.7.1 Python/3.7.12 (Windows-10-10.0.19041-SP0)
+      - azsdk-python-identity/1.10.0 Python/3.7.8 (Windows-10-10.0.19041-SP0)
```

Comment:
It seems that the recordings were generated this time using Python 3.7.8 which is a downgrade from 3.7.10 in the previous ones? Is this intentional? I don't think we need to regenerate them for this but wanted to double check.

Reply:
That's the version I have installed. I should probably update my environement, lol.